capstone/src/main/java/com/google/univiz/api/dataviz/CollegeDataStatsConverter.java

```diff
@@ -0,0 +1,26 @@
+package com.google.univiz.api.dataviz;
```

Comment:
Since #77, i'm not sure what is the right place for this class and in general Converter<> classes. I feel like univiz.api.resource is the right place. It'll be next to the `*Impl.java` class that is going to use it.